ansible/hadoop.yml

```diff
@@ -15,12 +15,30 @@
 # limitations under the License.
 #
 
-- hosts: all
+- hosts: all:!{{ azure_proxy_host }}
   roles:
     - hadoop
+- hosts: journalnode
+  tasks:
+    - import_tasks: roles/hadoop/tasks/start-journal.yml
+- hosts: namenode[0]
+  tasks:
+    - import_tasks: roles/hadoop/tasks/format-nn.yml
+- hosts: namenode[0]
+  tasks:
+    - import_tasks: roles/hadoop/tasks/format-zk.yml
 - hosts: namenode
   tasks:
-    - import_tasks: roles/hadoop/tasks/start-hdfs.yml
+    - import_tasks: roles/hadoop/tasks/start-zkfc.yml
+- hosts: namenode[0]
+  tasks:
+    - import_tasks: roles/hadoop/tasks/start-nn1.yml
+- hosts: namenode[1]
```

Comment:
What if the user only configured a single namenode?

Reply:
@keith-turner Currently the setup assumes that user must configure Namenode HA to eliminate Namenode as a potential Single Point of Failure.

Reply:
I don't think that's a good setup assumption. This tool is a developer tool, intended for testing Fluo (and Accumulo) by its contributors. It is not a tool for public release, and is not intended as a cluster provisioning tool for production environments. HA is not necessarily worthwhile for the use case Muchos was written for.

Reply:
Personally I only use Muchos for testing Accumulo and Fluo releases.

Reply:
I resolved this because it overlaps with another conversation were its being discussed to make HA an optional feature as follow on work.